src/components/StatsScoringDialog.vue

```diff
@@ -2,10 +2,12 @@
   <v-dialog v-model="show">
     <!-- Activator -->
     <template #activator="{ props }">
-      <v-btn color="primary" class="mb-2" v-bind="props" variant="plain">
-        <span v-if="showButtonText">How are ranks determined?</span>
-        <v-icon class="ml-1" icon="mdi-information-outline" />
-      </v-btn>
+      <slot name="button">
+        <v-btn color="primary" class="mb-2" v-bind="props" variant="plain">
+          <span v-if="showButtonText">How are ranks determined?</span>
```

Comment:
The use of this prop feels redundant with the slot approach; I think we want one or the other, since both are about what goes into the button used to open the same dialog. I'd be to go either way, but I think if we should either nix the prop and make this text appear unless you override the slot, or make it entirely prop-driven and nix the slot.

Funnily enough, the main driver for the use of the slot was getting the `data-cy` attribute to drill down properly, right? The goofy thing is that there isn't really a need (as I see it) for the consumers of this component to have different data-cy selectors! We could just put the data-cy selector into this component itself, directly on the button, and then there's no need for the parent + child prop combining.

The only thing that changes functionally is that all consumers (2) of the component will have the same data-cy attribute on its activator, but I think that's actually fine. It's one component, just slightly customized so we can select it the same way. What do you think?

Reply:
Good idea, I'll switch to just using the prop and move the cy selector down. I probably should have spent more time looking in to the why, but figured that's better for post "everything working" so we have time to rethink architecture. Mostly in "get it working" mode at the moment.

This is a simple change that I'm in favor of though, since it'll reduce complexity of the implementation. Good stuff! 👍🏼